packages/angular_devkit/schematics/bin/schematics.ts

```diff
@@ -154,7 +154,7 @@ if (collection === null) {
 
 /** If the user wants to list schematics, we simply show all the schematic names. */
 if (argv['list-schematics']) {
-  logger.info(engineHost.listSchematics(collection).join('\n'));
+  logger.info(engine.listSchematicNames(collection).join('\n'));
```

Comment:
This should be part of the 2nd commit in the PR.

Reply:
Oh right. Updated.